ckanext/switzerland/templates/header_ogdch.html

```diff
@@ -34,7 +34,7 @@
             ('organizations_index', _('Organizations')),
           ) }}
           <li>
-            <a href="{{ h.url('/app') }}">{{ _('Applications') }}</a>
+            <a href="{{ h.url_for('/showcase') }}">{{ _('Applications') }}</a>
```

Comment:
AFAIK it is preferred to use the controller in the `url_for` and build the url like they do here:
https://github.com/ckan/ckanext-showcase/blob/master/ckanext/showcase/controller.py#L241-L243

See also:
https://docs.ckan.org/en/2.8/theming/template-helper-functions.html#ckan.lib.helpers.url_for

The way it is done here is still possible but deprecated, when we do have a controller we can link to

Reply:
In this case I don't want the Controller and the action in the url, because I want the url to be the same, regardless whether I am coming from the WP side of the site or the CKAN side.

If I use what you suggest, then I get urls such as `http://ogdch.test/fr/showcase?action=showcase_list&controller=ckanext.showcase.controller%3AShowcaseController` 
when I am coming from `http://ogdch.test/fr/dataset`  which is not a memorable url and not the url that I get when I click on Applications from the front page, which gets me to `http://ogdch.test/fr/showcase`

Therefore I want to stick to this suggested version for now. Would that be okay for you?

Reply:
Okay, thank you for your explanation!
We have to keep those cases in mind, when we refactor the deprecated routes in the future but this is fine :)